cql3/untyped_result_set.cc

```diff
@@ -69,25 +69,32 @@ bool cql3::untyped_result_set_row::has(const sstring& name) const {
 
 using cql_transport::messages::result_message;
 
+cql3::untyped_result_set::untyped_result_set(const cql3::result_set& rs) {
+    auto& cn = rs.get_metadata().get_names();
+    for (auto& r : rs.rows()) {
+        _rows.emplace_back(cn, std::move(r));
```

Comment:
cn == column_names. As in the result of "get_names()". As in a map name->column index. And yes, right now this is duplicated for each row. Because untyped res-set was initially meant for more or less single-row, cold code, usage. 
Making this type more efficient is on the todo list.